model/clusters_mgmt/v1/cluster_api_type.model

```diff
@@ -20,4 +20,24 @@ struct ClusterAPI {
 	URL String
 	// The listening method of the API server.
 	Listening ListeningMethod
+	
+	// Describes the access policy for CIDR Blocks to the Kubernetes API server
+	CIDRBlocksAccess CIDRBlocksAccess
```

Comment:
I noticed we have a "network" struct on the cluster, that have some related cidr settings. I assume the choice to put it under "api" is because it's relevant ONLY for kube-api-server access. @miguelsorianod @nimrodshn is that correct?

Reply:
Yes, correct.